src/common/index.jsx

```diff
@@ -28,17 +28,18 @@ export class HotTable extends React.Component {
   constructor() {
     super();
 
-    this.hotInstance = null;
     this.settingsMapper = new SettingsMapper();
     this.id = null;
+    this.hotInstance = null;
+    this.hotElementRef = React.createRef();
```

Comment:
`React.createRef()` was introduced in v16.3: https://reactjs.org/blog/2018/03/29/react-v-16-3.html#createref-api Which was released quite recently.

I think we should stick to callback refs: https://reactjs.org/docs/refs-and-the-dom.html#callback-refs but without using them inline. Reason to use it as a class method: https://reactjs.org/docs/refs-and-the-dom.html#caveats-with-callback-refs . We can keep backward compatibility with older React versions then. What to you think?

Reply:
You're probably right.